examples/load-reporting-service/server/lrs_server.go

```diff
@@ -0,0 +1,161 @@
+package server
```

Comment:
I think this file could use *a lot* more comments, especially given that it's an example